101-automation-with-modules-and-runbook-execution/azuredeploy.parameters.json

```diff
@@ -0,0 +1,27 @@
+{
+  "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "UserEmail": {
+      "value": "ps.5026@gmail.com"
+    },
+    "Password": {
+      "value": "GEN-PASSWORD"
+    },
+    "WebAppName": {
+      "value": "MyWebApp"
```

Comment:
will need to use prereq values for this

Reply:
I will include this with website deployment template

Reply:
removed it for now